lib/person.rb

```diff
@@ -0,0 +1,36 @@
+require 'date'
+require 'pry'
+
+class Person
+  attr_accessor :portfolio
+
+  def initialize
+    @portfolio = []
+  end
+
+  def rent(library, args={})
+    #binding.pry
+    item = library.books.detect { |obj| obj[args.keys.first] == args[args.keys.first] }
+    to_add = {}
+    if item
+      if args.keys.first == :title
+        library.lend(self, title: item[:title])
+      elsif args.keys.first == :author
+        library.lend(self, author: item[:author])
+      end
+      to_add.merge!({title: item[:title],
```

Comment:
Performance/RedundantMerge: Use `to_add[:title] = item[:title]; to_add[:author] = item[:author]; to_add[:duedate] = duedate` instead of `to_add.merge!({title: item[:title],
                     author: item[:author],
                     duedate: duedate})`. (https://github.com/JuanitoFatas/fast-ruby#hashmerge-vs-hash-code)<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.